dist/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Development</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1"><script defer src="/index.bundle.js"></script></head>
+  <body>
+  </body>
```

Comment:
Pour permettre l'init de react-dom (après ajout de `src/index.js`)
```suggestion
  <meta name="viewport" content="width=device-width, initial-scale=1"><script defer src="/index.bundle.js"></script></head>
  <body>
    <div id="app></div>
  </body>
```

Reply:
Au départ c'est ce que j'avais fait. Mais à chaque build, la div avec l'ID s'enlève.

Reply:
Essaie avec le script en fin de body. Même avec le defer ça prend pas toujours